app/src/main/java/com/willowtree/vocable/settings/EditPresetsFragment.kt

```diff
@@ -130,6 +130,9 @@ class EditPresetsFragment : BaseFragment() {
                     phrasesAdapter.setPhrases(phrases)
                     // Move adapter to middle so user can scroll both directions
                     val middle = phrasesAdapter.itemCount / 2
+                    if(phrasesAdapter.numPages == 0) {
```

Comment:
Which cases have a `numPages` of 0?

Reply:
when you delete all the phrases in My Sayings

Reply:
wasn't sure how to handle this, but this works :P

Reply:
Gotcha 👍